Sources/RxAnimated/RxAnimated+animations.swift

```diff
@@ -1,5 +1,7 @@
 import RxSwift
 import RxCocoa
+#if canImport(UIKit)
```

Comment:
If in Package.swift we specify platforms do we need to do this `canImport(UIKit)` here?